libgtpv2c/src/gtp_ies_decoder.c

```diff
@@ -371,7 +371,7 @@ int decode_gtp_prot_cfg_opts_ie(uint8_t *buf,
     uint16_t decoded = 0;
     value->pco = decode_bits(buf, total_decoded, 8, &decoded);
     total_decoded += decoded;
-    return total_decoded/CHAR_SIZE;
+	return value->header.len + IE_HEADER_SIZE;
```

Comment:
PCO decoding does not seem to be correct. PCO is 256 byte array length field.  currently we are just copying 1 byte. Same applies to other IEs.

Reply:
The correct structure and encoding/decoding of PCO, APCO will be part of separate libgtpv2c project.